pkg/resourcemonitor/resourcemonitor.go

```diff
@@ -46,10 +46,10 @@ type ResourceExcludeList struct {
 }
 
 type Args struct {
-	Namespace          string
-	SysfsRoot          string
-	ExcludeList        ResourceExcludeList
-	RefreshAllocatable bool
+	Namespace            string
+	SysfsRoot            string
+	ExcludeList          ResourceExcludeList
+	RefreshNodeResources bool
```

Comment:
it's very debatable that this change breaks the commandline tests (`main_test.go`), but let's address on a different PR.